src/scripts/jefferson

```diff
@@ -318,21 +384,42 @@ def scan_fs(content, endianness, verbose=False):
                     inode.unpack(content_mv[0 + offset :])
 
                     if inode.ino in fs[JFFS2_NODETYPE_INODE]:
-                        if inode.version > fs[JFFS2_NODETYPE_INODE][inode.ino].version:
-                            fs[JFFS2_NODETYPE_INODE][inode.ino] = inode
+                        fs[JFFS2_NODETYPE_INODE][inode.ino].append(inode)
                     else:
-                        fs[JFFS2_NODETYPE_INODE][inode.ino] = inode
+                        fs[JFFS2_NODETYPE_INODE][inode.ino] = [inode]
                     if verbose:
                         print("0x%08X:" % (offset), inode)
-                elif unknown_node.nodetype == JFFS2_NODETYPE_CLEANMARKER:
-                    pass
-                elif unknown_node.nodetype == JFFS2_NODETYPE_PADDING:
-                    pass
                 elif unknown_node.nodetype == JFFS2_NODETYPE_SUMMARY:
-                    pass
+                    summary = Jffs2_raw_summary()
+                    summary.unpack(content_mv[0 + offset :])
+                    fs[JFFS2_NODETYPE_SUMMARY].append(summary)
+                    if verbose:
+                        print("0x%08X:" % (offset), summary)
```

Comment:
Not needed because those node types are not used to reconstruct the filesystem on disk. We parse structures that are never used afterwards.